src/api/app/views/webui/users/tokens/hide.js.erb

```diff
@@ -0,0 +1,3 @@
+$('#token-reveal').show();
+$('#token-hide').addClass('d-none');
+$('#token-secret').empty();
```

Comment:
This could have been done by plain js. Why add ajax?

Reply:
I was thinking that using javascript increased security of the thing because you had to authenticate on clicking the reveal button, but now I think that if we go for the simple plan js, you need to be authenticated as well...
Let me try

Reply:
Fixed. Please, review again!